internal/server/rewrapping_test.go

```diff
@@ -45,12 +45,12 @@ func TestRewrap_workerAuthCertRewrapFn(t *testing.T) {
 	assert.NoError(t, err)
 
 	// decrypt with the new key version and check to make sure things match
-	decryptedGotPrivKey, err := decrypt(ctx, got.GetPrivateKey(), kmsWrapper2)
+	decryptedGotPrivKey, err := decrypt(ctx, got.GetCtPrivateKey(), kmsWrapper2)
```

Comment:
Should we eventually be able to remove these `encrypt` `decrypt`methods altogether?

Reply:
yes, but I was hesitant to do so immediately so that we're not using the encrypt/decrypt functions to test to make sure the encrypt/decrypt functions work. leaving the test with these funcs gives us an end to end test to make sure they're working as intended

Reply:
hm, can we define the functions in the test file then so they're inaccessible from the other package?

Reply:
I think this depends on if we feel encrypt/decrypt func tests are necessary. I don't see them anywhere else in the codebase (maybe because its mostly handled by structwrapping?) if that's the case we can just switch this to use the worker auth decrypt and potentially remove the other funcs

Reply:
I would like to make it so that using the `.encrypt` and `.decrypt` methods is the obvious (ideally only) way to do this operation. Leaving these functions behind may mean someone uses them in the future, which means we'll have multiple ways of doing a single operation, which is error prone 😬.